roles/openshift_gce_cluster/templates/provision.j2.sh

```diff
@@ -0,0 +1,318 @@
+#!/bin/bash
```

Comment:
Does ansible support gce provisioning?

Reply:
There is some limited support, but it's a mix.  As mentioned above gcloud is the public API for these functions, and over the medium term these should be moved to deployment manager.